### ISSUE TITLE
Support to mesh networks

### ISSUE THREAD

**Issue by Plasmmer** [ZeroNetJS/zeronet-js#18]

**Comment by mkg20001** [comment]
ToDos:
 - [x] mDNS
 - [x] Circuit Relay
 - [ ] DHT
 - [ ] [zeronet-tracker](https://github.com/ZeroNetJS/zeronet-tracker)
 - [ ] Tracker relay

**Comment by Plasmmer** [comment]
Ethernet networks also supported?

**Comment by mkg20001** [comment]
Yes using mDNS it should work but it might be a bit buggy... (as it hasn't been tested extensively)
